minecraft/minecraft-test/build.gradle

```diff
@@ -11,6 +11,16 @@ loom {
 
 def minecraft_version = "1.19.3";
 
+configurations {
+	testAnnotationProcessor {
+		extendsFrom annotationProcessor
+	}
```

Comment:
Is this needed? Presumably it's for mixins but there are none in the test source set

Reply:
No, this is a left over from some mess I had before. Removed 👍